homepage/homepage/app/docs/[framework]/[...slug]/project-setup/svelte.mdx

```diff
@@ -0,0 +1,96 @@
+import { CodeGroup } from "@/components/forMdx";
+
+# Svelte Installation
+
+Jazz can be used with Svelte or in a SvelteKit app.
+
+To add some Jazz to your Svelte app, you can use the following steps:
+
+1. Install Jazz dependencies
+
+<CodeGroup>
+```sh
+pnpm install jazz-tools jazz-svelte
+```
+</CodeGroup>
+
+2. Write your schema
+
+See the [schema docs](/docs/react/schemas/covalues) for more information.
+
+<CodeGroup>
+```ts
+// src/lib/schema.ts
+import { Account, Profile, co } from "jazz-tools";
+
+export class MyProfile extends Profile {
+  name = co.string;
+  counter = co.number;
+}
```

Comment:
Small catch on the profile, the data can be read by everyone.

Maybe we should add a small comment about it?

Reply:
Yeah I thought about and figured it's just a demo, but you're right a comment would help

Reply:
Added a note